packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -318,106 +317,38 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
-
-                            this._getLogger(`'before remote scope'`),
-                            this._getLogger(
-                              `'globalThis.__remote_scope__'`,
-                              `globalThis.__remote_scope__`
-                            ),
-                            this._getLogger(
-                              `'globalThis.__remote_scope__[${JSON.stringify(
-                                name
-                              )}]'`,
-                              `globalThis.__remote_scope__[${JSON.stringify(
-                                name
-                              )}]`
-                            ),
 
-                            /*   TODO: this global.REMOTE_CONFIG doesnt work in this v5 core, not sure if i need to keep it or not
-                         not deleting it yet since i might need this for tracking all the remote entries across systems
-                         for now, im going to use locally known remote scope from remoteEntry config
-                         update: We will most likely need this, since remote would not have its own config
-                         id need to access the host system and find the known url
-                          basically this is how i create publicPath: auto on servers.
-                         `var requestedRemote = global.REMOTE_CONFIG[${JSON.stringify(
-                          name
-                        )}]`,
-                     */
-                            `var requestedRemote = remoteRegistry[${JSON.stringify(
-                              name
-                            )}]`,
+                            `var requestedRemote = ${
+                              RuntimeGlobals.require
+                            }.federation.remotes[${JSON.stringify(name)}]`,
                             this._getLogger(
                               `'requested remote'`,
                               `requestedRemote`
                             ),
-                            /*TODO: we need to support when user implements own promise new promise function
-                            for example i have my own promise remotes, not global@remotename
-                            so there could be cases where remote may be function still - not sure */
 
-                            /*TODO: need to handle if chunk fetch fails/crashes - ensure server still can keep loading
-                            right now if you throw an error in here, server will stall forever */
+                            /*TODO:
+                            need to handle if chunk fetch fails/crashes - ensure server still can keep loading
+                            right now if you throw an error in here, server will stall forever
+                            */
 
                             "if(typeof requestedRemote === 'function'){",
                             Template.indent(
                               'requestedRemote = await requestedRemote()'
```

Comment:
Sometimes remote can be promise function of derived at runtime. Should allow await to resolve remote entry url and resister it in system